src/pages/Login.vue

```diff
@@ -395,16 +397,32 @@ export default {
       // whether the phone number entered by the user is valid
       return this.phoneInput.toString().match(/^([0]|\+91)?[6-9]\d{9}$/g) != null;
     },
+    startCountdown(seconds) {
+      // counts seconds before enabling resend OTP button

```

Comment:
we have not updated the docstrings in this file yet but refer to the function `addNewItem` in `Editor.vue` for how we are writing docstrings for functions.